src/otx/recipe/instance_segmentation/openvino_model.yaml

```diff
@@ -35,4 +35,4 @@ overrides:
       image_color_channel: RGB
       data_format: coco_instances
       test_subset:
-        batch_size: 2
+        batch_size: 256
```

Comment:
Isn't it too large for test case?

Reply:
I reduced it to 64 the same as in detection.